scripts/perftest/README.md

```diff
@@ -28,10 +28,16 @@ There are a few prerequisites:
 - Setup OpenBlas: <https://github.com/xianyi/OpenBLAS/wiki/Precompiled-installation-packages>
 - Install Perf stat: <https://linoxide.com/linux-how-to/install-perf-tool-centos-ubuntu/>
 
-NOTE THE SCRIPT HAS TO BE RUN FROM THE ROOT OF THE REPOSITORY.
+NOTE THE SCRIPT HAS TO BE RUN FROM THE PERFTEST FOLDER.
 
+Examples:
 ```bash
-./scripts/perftest/runAll.sh
+./runAll.sh
+
+./runAll.sh perftestTemp MR
```

Comment:
I don't think we need the MR vs CP differentiation here; simply let a developer specify the CMD that is used consistently.

Reply:
You mean so it can be executed like e.g. "./runAll.sh systemds" or "./runAll.sh ./sparkDML.sh"?

Reply:
Lets say I understood it correctly and "./runAll.sh ./sparkDML.sh" is a valid call, then the problem here is that the sparkDML.sh script needs double dashes for its flag arguments I believe.
Should I do a substring check if the argument contains "sparkDML.sh" and then prepend an additional dash to the flags?

Reply:
Sorry for the confusion - I think simply defining `CMD="./sparkDML.sh"` in `runAll.sh` would suffice - that way people can simply edit this once location if they want to run via systemds and everything becomes less convoluted (no more need to handle this in each script).